File: 3379-score-of-a-string/3379-score-of-a-string.js
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function(s) {
    let arr= [];
    for(i = 0; i<s.length-1;i++){
        let a = s.charCodeAt(i);
        let b = s.charCodeAt(i+1);
        let c = a-b;
        arr.push(Math.abs(c))
    }
    return arr.reduce((ac,val)=>ac+val);
};

File: 3379-score-of-a-string/3379-score-of-a-string.cs
public class Solution {
    public int ScoreOfString(string s) {
        int val = 0;
        char k = s[0];
        int prev=(int)k;
        foreach(char i in s){
            
            val+= Math.Abs(prev-(int)i);
            prev = (int)i;
        
        }
        return val;
    }
}